Makefile

```diff
@@ -19,3 +19,34 @@ test:
 
 test-coverage:
 	pytest --cov=task_manager --cov-report xml --cov-report=html
+
+start-postgres:
+	@docker-compose up -d postgres
```

Comment:
hmmmmm, I see no `docker-compose.yml` file included in this PR 🤔 was it left out by any chance?